src/main.cpp

```diff
@@ -272,6 +272,11 @@ main(int argc, char *argv[]) {
 
   reed_solomon_init();
   auto input_deinit_guard = input::init();
+
+  if (input::probe_gamepads()) {
```

Comment:
Do we re-probe before starting a session like we do for encoders?

Reply:
@ReenigneArcher Not right now, just on start, but in another PR I want dynamically fetch this on the Web UI and show the current status to the user, I could probe on session too. Do you want the session probe in this PR or I could postpone?

Reply:
I think it can be later as I don't think it's any worse than what we're doing now (and probably not as important as encoder probing).